docs/fundamentals/networking/telemetry/tracing.md

```diff
@@ -0,0 +1,169 @@
+
+The steps to use *ServiceDefaults* outside .NET Aspire are:
+
+- Add the *ServiceDefaults* project to the solution using Add New Project in Visual Studio, or use `dotnet new aspire-servicedefaults --output ServiceDefaults`
+- Reference the *ServiceDefaults* project from your ASP.NET application. In Visual Studio use "Add -> Project Reference" and select the *ServiceDefaults* project"
+- Call its OpenTelemetry setup function as part of your application builder initialization.
+
+``` csharp
+var builder = WebApplication.CreateBuilder(args);
+builder.ConfigureOpenTelemetry();
+
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+app.Run();
+```
+
+For a full walkthrough, see [Example: Use OpenTelemetry with OTLP and the standalone Aspire Dashboard](../../../core/diagnostics/observability-otlp-example.md).
+
+
+### Collecting traces manually
+
+For a walkthrough of how to collect distributed traces, as well as metrics without using Aspire Service Defaults, see  [Example: Use OpenTelemetry with Prometheus, Grafana, and Jaeger](../../../core/diagnostics/observability-prgrja-example.md). 
+
+## Experimental connection spans in .NET 9
+
+.NET 9 adds a handful of new spans for collecting detailed connection information:
+
+| Activity Source | Description |
+| --- | --- |
+| Experimental.System.Net.NameResolution | Tracks DNS resolution for anything using the .NET DNS api's such as HttpClient |
+| Experimental.System.Net.Sockets | Tracks socket connection activity |
+| Experimental.System.Net.Security | Tracks TLS handshake for inbound and outbound connections |
+| Experimental.System.Net.Http.Connections | Tracks the Connection pool for HttpClient |
+
+These spans are available starting with .NET 9. The ActivitySource Names start with `Experimental` as these spans are not yet included in the OpenTelemetry Semantic conventions, and may be changed as we learn more about how well they work in production. 
+
+These spans are probably too verbose for use 24x7 in production scenarios with high workloads - they are somewhat noisy and this level of data is not normally needed. However if you are trying to diagnose connection issues or get a deeper understanding of how network and connection latency is affecting your services, then they provide insight that is hard to collect by other means. 
+
+Note: When enabled, the http connection span is linked to from HttpClient request spans. As an http connection can be long lived, this could result in many links to the connection span from each of the request spans. Some APM monitoring tools aggresively walk links between spans to build up their views and so including this span may cause issues when the tools were not designed to account for large numbers of links.
```

Comment:
```suggestion
> [!NOTE]
> When enabled, the http connection span is linked to from HttpClient request spans. As an http connection can be long lived, this could result in many links to the connection span from each of the request spans. Some APM monitoring tools aggresively walk links between spans to build up their views and so including this span may cause issues when the tools were not designed to account for large numbers of links.
```

Moreover, it would make sense to draw attention to the need to upgrade to .NET Aspire 9.0 (eg. via Upgrade Assistant) since Span Links are not supported in 8.0.